src/routes/(app)/order/[id]/fetchOrderForUser.ts

```diff
@@ -110,7 +123,11 @@ export async function fetchOrderForUser(orderId: string, params?: { userRoleId?:
 		payments: order.payments.map((payment) => ({
 			id: payment._id.toString(),
 			method: payment.method,
+			posSubtype: payment.posSubtype,
 			posTapToPay: payment.posTapToPay,
+			tapToPayOnActivationUrl: payment.posSubtype 
+				? (posSubtypesMap.get(payment.posSubtype)?.tapToPayOnActivationUrl ?? runtimeConfig.posTapToPay.onActivationUrl)
+				: runtimeConfig.posTapToPay.onActivationUrl,
```

Comment:
I think this is problematic:

- The UI to configure RuntimeConfig item `posTapToPay` no-longer exists.
- `onActivationUrl` was configured before this patchset, there's no way of removing the value.
- We have no reason to believe the tap-to-pay URL configured before this patchset is a reasonable default.